res/css/views/voip/_VideoLobby.scss

```diff
@@ -0,0 +1,161 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_VideoLobby {
+    height: 100%;
+    padding: 12px;
+
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 44px;
+
+    .mx_VideoLobby_preview {
+        position: relative;
+        width: 100%;
+        max-width: 800px;
+        aspect-ratio: 1.5;
+        background-color: $system;
+
+        border-radius: 20px;
+        overflow: hidden;
+
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+
+        .mx_BaseAvatar {
+            margin: 20px;
+
+            // Responsive sizing
+            width: unset !important;
+            height: unset !important;
```

Comment:
Whenever we use `!important` we should be documenting what exactly we're overriding.